src/tasks/components/TaskCard.tsx

```diff
@@ -145,15 +145,13 @@ export class TaskCard extends PureComponent<
 
   private handleNameClick = (event: MouseEvent) => {
     const {
+      history,
+      task,
       match: {
         params: {orgID},
       },
-      history,
-      task,
-      setCurrentTasksPage,
     } = this.props
-    const url = `/orgs/${orgID}/tasks/${task.id}/edit`
-    setCurrentTasksPage(TaskPage.TasksPage)
+    const url = `/orgs/${orgID}/tasks/${task.id}/runs`
 
     if (event.metaKey) {
       window.open(url, '_blank')
```

Comment:
I think we want to do something like this:
```suggestion
      window.open(url, '_blank', 'noopener')
```

it's a security issue https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener